pom.xml

```diff
@@ -305,5 +307,61 @@
        </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
Agreed, this will show up in a new PR.
Issue #7 has [many of these updates](https://github.com/GoogleCloudPlatform/jetty-runtime/blob/issue-7/pom.xml#L311-L314) already.
This PR is holding up too many other PRs at the moment.